fhir-persistence/src/test/java/com/ibm/fhir/persistence/test/common/AbstractPersistenceTest.java

```diff
@@ -175,8 +175,13 @@ public void tearDown() throws Exception {
                     // ignore the chained part and just verify the reference param is there
                     paramName = key.split("\\.")[0];
                 }
-                // strip any modifiers
-                final String finalParamName = paramName.split(":")[0];
+                // strip any modifiers, unless :of-type
+                final String finalParamName;
+                if (key.endsWith(":of-type")) {
```

Comment:
':identifier' modifier as well?  See other comment in JDBCParameterBuildingVisitor.java.

Reply:
no updates required here